app/models/project.rb

```diff
@@ -1,4 +1,11 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def avg_exp
+    return 0 if contestants.count == 0 
+    (contestants.sum { |contestant| contestant.years_of_experience }.to_f / contestants.count).round(1)
```

Comment:
If you find yourself doing math with Ruby in a model method, it's usually a good indication that there's an AR method that could handle this for you more easily! Consider:
`contestants.average(:years_of_experience).to_f`